src/plus/launchpad/launchpadProvider.ts

```diff
@@ -210,44 +214,74 @@ export class LaunchpadProvider implements Disposable {
 	}
 
 	private async getSearchedPullRequests(search: string, cancellation?: CancellationToken) {
-		// TODO: This needs to be generalized to work outside of GitHub,
-		// The current idea is that we should iterate the connected integrations and apply their parsing.
-		// Probably we even want to build a map like this: { integrationId: identity }
-		// Then we iterate connected integrations and search in each of them with the corresponding identity.
-		const { ownerAndRepo, prNumber, provider } = getPullRequestIdentityFromMaybeUrl(search);
-		let result: TimedResult<SearchedPullRequest[] | undefined> | undefined;
-
-		if (provider != null && prNumber != null && ownerAndRepo != null) {
-			// TODO: This needs to be generalized to work outside of GitHub/GitLab
-			const integration = await this.container.integrations.get(provider);
-			const [owner, repo] = ownerAndRepo.split('/', 2);
-			const descriptor: GitHubRepositoryDescriptor | GitLabRepositoryDescriptor = {
-				key: ownerAndRepo,
-				owner: owner,
-				name: repo,
-			};
-			const pr = await withDurationAndSlowEventOnTimeout(
-				integration?.getPullRequest(descriptor, prNumber),
-				'getPullRequest',
-				this.container,
-			);
-			if (pr?.value != null) {
-				result = { value: [{ pullRequest: pr.value, reasons: [] }], duration: pr.duration };
-				return { prs: result, suggestionCounts: undefined };
+		const prUrlIdentity: PullRequestUrlIdentity | undefined = await this.getPullRequestIdentityFromMaybeUrl(search);
+		const result: { readonly value: SearchedPullRequest[]; duration: number } = {
+			value: [],
+			duration: 0,
+		};
+
+		const connectedIntegrations = await this.getConnectedIntegrations();
```

Comment:
If we're already getting the connected integrations below, maybe we can just pass them into `this.getPullRequestIdentityFromMaybeUrl`

Reply:
fixed